### ISSUE TITLE
bang tolong

### ISSUE THREAD

**Issue by FahriAdison** [stegripe/bajigur#684]
> bajigur@1.5.5 start
> node --env-file=.env .
node: bad option: --env-file=.env

**Comment by Mednoob** [comment]
Versi node yang mas pakai versi berapa?

**Comment by FahriAdison** [comment]
18

**Comment by FahriAdison** [comment]
Harus 20 kah

**Comment by FahriAdison** [comment]
gw udah pindahkan ke versi 20 tapi ada lagi ini,gw bingung
node:internal/modules/cjs/loader:451
      throw err;
      ^
Error: Cannot find module '/home/container/dist'. Please verify that the package.json has a valid "main" entry
    at tryPackage (node:internal/modules/cjs/loader:443:19)
    at Module._findPath (node:internal/modules/cjs/loader:714:18)
    at resolveMainPath (node:internal/modules/run_main:26:23)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:128:24)
    at node:internal/main/run_main_module:28:49 {
  code: 'MODULE_NOT_FOUND',
  path: '/home/container/package.json',
  requestPath: '/home/container'
}

**Comment by mzrtamp** [comment]
Itu pake pterodactyl? Hapus semua isi servernya terus install ulang di node 20

**Comment by FahriAdison** [comment]
startup nya apa bang,npm start atau pnpm start atau sama keduanya?

**Comment by mzrtamp** [comment]
Coba liat yang punyamu sekarang

**Comment by FahriAdison** [comment]
![WhatsApp Image 2024-04-25 at 17 13 31](https://github.com/stegripe/bajigur/assets/92633211/f1eebba4-60a7-4882-ae65-ddb17614b6b5)

**Comment by Mednoob** [comment]
Belum di build itu sama masnya projectnya. Coba di set `CMD_RUN` nya ke `npm run build && /usr/local/bin/npm run start`

**Comment by FahriAdison** [comment]
maksudnya gimana bang,aku belum ngerti hehe, soalnya itu di panel udah lengkap sama egg nya,yang aku bingung itu pakai npm start atau pnpm start atau itu keduanya,start nya udah ada

**Comment by Mednoob** [comment]
Iya, saya tau itu udah lengkap sama eggnya. Cuma, masalahnya di sini, masnya gak bisa langsung `npm start` atau `pnpm start`. Projectnya ini harus di compile terlebih dahulu dari TypeScript ke JavaScript. Setelah itu baru bisa di `npm start`.

Nah, tadi saya suruh set `CMD_RUN` nya ke itu biar dia nya compile dulu.
